sql/search_derived/search_metric_contribution/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+friendly_name: Search Metric Contribution
+description: >
+  A table shows how much of one metric is contributed by another metric.
+owners:
+  - 
+labels:
+  schedule: daily
+scheduling:
+  dag_name: 
```

Comment:
```suggestion
  dag_name: bqetL_search
```

Reply:
Thanks for all the comments. I will review these and get back to you!